uui/components/filters/FilterPanelItemToggler.tsx

```diff
@@ -42,10 +42,10 @@ export const FilterPanelItemToggler = React.forwardRef<HTMLDivElement, FilterToo
             ref={ ref }
         >
             <FlexRow cx={ css.titleWrapper }>
-                <Text color="gray60" cx={ css.title }>{ getTitle }</Text>
+                <Text color="secondary" cx={ css.title }>{ getTitle }</Text>
```

Comment:
use specific filter panel tokens instead of semantic text

Reply:
done